app/views/curation_concerns/single_use_links/new_show.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="single-use-link">
+<h1>Single Use Link</h1>
+<p>Anyone can use the following link once to view the file metadata</p>
+<%= link_to @asset, @link %>
```

Comment:
@cam156 for some reason this link also gives an error without including `target: '_blank'` when using the presenter class.